roles/commons/tasks/is_monitored.yml

```diff
@@ -15,7 +15,7 @@
     - monitoring
     - nrpe_checks
   template: src=nrpe.cfg.j2
-            dest={{ nrpe_conf_path }}/nrpe.cfg backup=yes
+            dest={{ nrpe_conf_path }}nrpe.cfg backup=yes
```

Comment:
Better let the nrpe_conf_path without the trailing slash and add the slash here after the var value (eg: {{nrpe_conf_path}}/nrpe.cfg

Reply:
Thanks for your review. 
I did it because I think it looks better.  But ok no problem.